### ISSUE TITLE
[MINIPLUS] [EQUALIZER] Equalizer does not open on GKD mini plus.

### ISSUE THREAD

**Issue by efylan** [game-de-it/plumOS-GKD#4]
When trying to open Equalizer on GKD mini plus. Shows splash screen then closes.

Thank you so much for the USB DAC mode!!!

**Comment by game-de-it** [comment]
Yes, the equalizer script only switches between ON and OFF, so nothing is displayed on the screen.

In order to let you feel the difference in sound caused by the equalizer, we have not displayed whether the equalizer is ON or OFF. Instead, you should listen to the sound and choose the one that suits you best.

When using USB-DAC or Bluetooth-AUDIO, the equalizer is bypassed and the raw sound is played from the USB-DAC.
This is because the sound tuning by the plumOS equalizer gets in the way of high-quality USB-DACs.

I like the moonlight2 USB-DAC, but I found a cheaper one with the same quality and sound quality.
https://ja.aliexpress.com/item/1005005785157157.html
I've been using this product recently. (^^)

I would be happy if you could tell me what USB-DAC you are using.

**Comment by efylan** [comment]
I'm using the xduoo x2s, it's pretty handy, no need to generate data that spotify or youtube should be paying us for, since it can play flac, ape and wav music on its own, too.

[https://ja.aliexpress.com/item/1005002388522831.html](https://ja.aliexpress.com/item/1005002388522831.html)

**Comment by game-de-it** [comment]
It's a good product.
It's convenient to be able to play music files in standalone mode.

I don't like the way that streaming on my phone wastes network and battery.

Japanese network companies have a limit on the amount of packets you can use, and if you exceed the limit, the bandwidth is reduced to 100Kbps.

It's very frustrating. lol

**Comment by efylan** [comment]
That sounds awful! 100kbps almost a like an old landline 52k modem connection...

I'm closing this issue as nonissue!
